src/content/docs/infrastructure/host-integrations/host-integrations-list/port-monitoring-integration.mdx

```diff
@@ -13,8 +13,90 @@ freshnessValidatedDate: never
 
 Our port monitoring integration monitors status for networking ports, such as TCP, UDP, etc., and reports data to New Relic.
 
-To get started, see the [Port monitoring on-host integration README](https://github.com/newrelic/nri-port-monitor) on GitHub.
-
 <Callout variant="tip">
   This integration is released as [open source on Github](https://github.com/newrelic/nri-port-monitor). A change log is also available there for the latest updates. Support for this integration is available with an [Expert Services subscription](https://www.newrelic.com/expertservices).
 </Callout>
+
+## Install the integration 
+
+Follow the steps below to install the integration:
+
+<Steps>
+    <Step>
+## Install the infrastructure agent [#infra-install]
+
+To use the Port Monitoring integration, you need to first [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) on the same host. All our on-host integrations require the infrastructure agent, which helps expose and report metrics to New Relic.  
+
+    </Step>
+    <Step>
+## Download and prepare the integration [#download]
+
+1. Download and unpack the ZIP file from [the GitHub repo](https://github.com/newrelic/nri-port-monitor/releases):
+
+    ```shell
+    wget https://github.com/newrelic/nri-port-monitor/releases/download/1.3/nri-port-monitor.tar.gz
```

Comment:
I notice we're having them download a specific version here. Maybe we can future proof this, like we're doing in the UNIX monitoring one